Makefile

```
@@ -13,18 +13,24 @@ MASTER_IMAGE = quay.io/dynatrace/dynatrace-operator:snapshot
 BRANCH_IMAGE = quay.io/dynatrace/dynatrace-operator:snapshot-$(shell git branch --show-current | sed "s/[^a-zA-Z0-9_-]/-/g")
 OLM_IMAGE ?= registry.connect.redhat.com/dynatrace/dynatrace-operator:v${VERSION}
 
-# "OTHERS" = ClusterRole, ClusterRoleBinding, Deployment, MutatingWebhookConfiguration, Role, RoleBinding, Service, ServiceAccount, ValidatingWebhookConfiguration
-KUBERNETES_OTHERS_YAML=config/deploy/kubernetes/kubernetes-others.yaml
-KUBERNETES_CRD_AND_OTHERS_YAML=config/deploy/kubernetes/kubernetes.yaml
-KUBERNETES_CSIDRIVER_YAML=config/deploy/kubernetes/kubernetes-csidriver.yaml
-KUBERNETES_OLM_YAML=config/deploy/kubernetes/kubernetes-olm.yaml
-KUBERNETES_ALL_YAML=config/deploy/kubernetes/kubernetes-all.yaml
-
-OPENSHIFT_OTHERS_YAML=config/deploy/openshift/openshift-others.yaml
-OPENSHIFT_CRD_AND_OTHERS_YAML=config/deploy/openshift/openshift.yaml
-OPENSHIFT_CSIDRIVER_YAML=config/deploy/openshift/openshift-csidriver.yaml
-OPENSHIFT_OLM_YAML=config/deploy/openshift/openshift-olm.yaml
-OPENSHIFT_ALL_YAML=config/deploy/openshift/openshift-all.yaml
+
+DYNATRACE_OPERATOR_CRD_YAML=dynatrace-operator-crd.yaml
+
+HELM_TEMPLATES_DIR=config/helm/chart/default/templates/
+HELM_CRD_DIR=Common/crd/
+
+MANIFESTS_DIR=config/deploy/
+
+# "COMPANIONS" = ClusterRole, ClusterRoleBinding, Deployment, MutatingWebhookConfiguration, Role, RoleBinding, Service, ServiceAccount, ValidatingWebhookConfiguration
+KUBERNETES_CRD_AND_COMPANIONS_YAML=$(MANIFESTS_DIR)/kubernetes/kubernetes.yaml
```

```suggestion
KUBERNETES_BASE_YAML=$(MANIFESTS_DIR)/kubernetes/kubernetes.yaml
```
`CRD_AND_COMPANIONS` says crd + stuff, which is not very descriptive yet still quite long.
 
`BASE` or `CORE` tells me that these are vital yamls, without which the operator will not work